modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -915,7 +956,7 @@
 
 //to ask if a host is willing
 /datum/action/cooldown/willing_host
-	name = "Willing Host (300 chemicals)"
+	name = "Willing Host (150 chemicals)"
```

Comment:
oh wowee, that is a huge buff.

Reply:
5 evolutions before you can get enough chems to even afford it means at least 125 brain damage and whatever trouble comes with that.